### ISSUE TITLE
[2019] Need to update speakers

### ISSUE THREAD

**Issue by gvegayon** [USCbiostats/rbootcamp#20]
Give credit where credit is due. Last year we had @EmilHvitfeldt @brookembell and @zhiiiyang (right @malcolmbarrett ?) I'll update the tag for last year's version.

**Comment by malcolmbarrett** [comment]
Yup! I can't remember if we had any other TAs in the course, but I believe only the folks listed here were speakers
